updating the readme file with note

sjyothi54 opened the pull request:
Hi @jaesius 
When I tried to integrate the snowflake integration in linux machine.I got the error like "$$NEWRELIC_SNOWFLAKE_HOME/snowflakeintegration-linux $$NEWRELIC_SNOWFLAKE_HOME/config.yaml $$NEWRELIC_SNOWFLAKE_HOME/queries/query_history.sql`
`/bin/sh: NEWRELIC_SNOWFLAKE_HOME/snowflakeintegration-linux: No such file or directory."
So I updated readme.md file with workaround.
This might fix some issues raised as well  in this repository.

matewilk commented:
Hi @sjyothi54 

Thanks so much for contributing.

I checked the PR and it seems like some parts of the edited README are not visible, specifically:

> **Note:** 
If you are getting error like,
 `$$NEWRELIC_SNOWFLAKE_HOME/snowflakeintegration-linux $$NEWRELIC_SNOWFLAKE_HOME/config.yaml $$NEWRELIC_SNOWFLAKE_HOME/queries/query_history.sql`
`/bin/sh: NEWRELIC_SNOWFLAKE_HOME/snowflakeintegration-linux: No such file or directory.`

Would you be able to fix it before I merge?

sjyothi54 commented:
@matewilk Thanks for the response. Could you please elaborate on what I need to fix?

nedl86 commented:
Hi @sjyothi54 thanks so much for your attention to this! I think there's an alternate way to get past this error, by adding the following to your agent configuration (`newrelic-infra.yml` file) and restarting the agent.

```
passthrough_environment:
  - NEWRELIC_SNOWFLAKE_HOME
```

I wonder if this may be a simpler fix to add to the README?

matewilk commented:
Hey @sjyothi54,

Sincere apologies for the late reply,
What I meant is that when you look at the `README.md` file in Github with your changes, the quoted text is not visible.
Is this intentional?
If not could you use [note](https://github.com/community/community/discussions/16925) notation instead?

sjyothi54 commented:
Sorry for the delayed response, @matewilk and @nedl86. 
I've implemented the recommendation from @nedl86, and it's functioning correctly on my end. Thank you for the advice. 
As a result, I have incorporated the modifications into this PR. Your review would be greatly appreciated. 
Thanks once again.

pkosanam reviewed (approved):
looks good to me.

Having this step on the readme file should help users in integrating snowflake with New Relic

pkosanam commented on README.md:
````diff
@@ -79,7 +79,14 @@ The integration relies on `NEWRELIC_SNOWFLAKE_HOME` environment variable to conn
+   ```
+   passthrough_environment:
+   - NEWRELIC_SNOWFLAKE_HOME
+   - SNOWSQL_ACCOUNT
````
```suggestion
passthrough_environment:
  - NEWRELIC_SNOWFLAKE_HOME
  - SNOWSQL_ACCOUNT
```
